proxy/src/main/java/com/wavefront/agent/PointHandlerImpl.java

```diff
@@ -49,19 +53,23 @@
   public PointHandlerImpl(final int port,
                           final String validationLevel,
                           final int blockedPointsPerBatch,
+                          final long discardPointsHours,
                           final PostPushDataTimedTask[] sendDataTasks) {
-    this(port, validationLevel, blockedPointsPerBatch, null, sendDataTasks);
+    this(port, validationLevel, blockedPointsPerBatch, null, discardPointsHours, sendDataTasks);
   }
 
   public PointHandlerImpl(final int port,
                           final String validationLevel,
                           final int blockedPointsPerBatch,
                           @Nullable final String prefix,
+                          final long discardPointsHours,
                           final PostPushDataTimedTask[] sendDataTasks) {
     this.validationLevel = validationLevel;
     this.port = port;
     this.blockedPointsPerBatch = blockedPointsPerBatch;
     this.prefix = prefix;
+    this.discardPointsHours = discardPointsHours;
+    millis_discard = DateUtils.MILLIS_PER_DAY * ((double) this.discardPointsHours/24);
```

Comment:
`this.millis_discard = DateUtils.MILLIS_PER_DAY * ((double) this.discardPointsHours / 24);`

`this` for consistency and spaces around `/`